packages/shared/src/components/post/PostWidgets.tsx

```diff
@@ -35,8 +35,7 @@ export function PostWidgets({
     if ('share' in navigator) {
       try {
         await navigator.share({
-          text: post.title,
-          url: post.commentsPermalink,
+          text: `${post.title} ${post.commentsPermalink}`,
```

Comment:
Not sure if this is preferred. Just a suggestion

```suggestion
          text: `${post.title}\n${post.commentsPermalink}`,
```